doc/source/security/data-at-rest-encryption.rst

```diff
@@ -0,0 +1,390 @@
+
+    +---------------+----------------+
+    | plugin_name   | plugin_status  |
+    +===============+================+
+    | keyring_file  | ACTIVE         |
+    +---------------+----------------+
+
+Encrypting a File-Per-Table Tablespace
+--------------------------------------
+
+The `CREATE TABLESPACE <https://dev.mysql.com/doc/refman/5.7/en/create-tablespace.html>`_ statement is extended to allow the ``ENCRYPTION=['Y/N']`` option to encrypt a File-per-Table tablespace.
+
+.. code-block:: mysql
+
+    mysql> CREATE TABLE myexample (id INT mytext varchar(255)) ENCRYPTION='Y';
+
+To enable encryption to an existing tablespace, add the ``ENCRYPTION`` option to the ``ALTER TABLE`` statement.
+
+.. code-block:: mysql
+
+    mysql> CREATE TABLE myexample ENCRYPTION='Y';
+
+You must add the ``ENCRYPTION`` option to ``ALTER TABLE`` to change the table encryption state. Without the ``ENCRYPTION`` option, an encrypted table remains encrypted or an unencrypted table remains unencrypted.
+
+Encrypting a General Tablespace
+-------------------------------------------
+
+As of :rn:`5.7.20-18`, |Percona Server| supports general tablespace encryption. A general tablespace must be either encrypted; all tables are encrypted, or not encrypted, no encrypted tables. You cannot encrypt only some of the tables in a general tablespace.
+
+.. rubric:: Automatically Encrypting Tablespaces
+
+Add the ``innodb_encrypt_tables`` variable to my.cnf to automatically encrypt general tablespaces. The possible values for the variable are:
+
+.. list-table::
+    :widths: 25 50
+    :header-rows: 1
+
+    * - Value
+      - Description
+    * - OFF
+      - The default value which disables automatic encryption of new tables
+    * - ON
+      - Enables automatic encryption for new tables
+    * - FORCE
+      - New tables are automatically created with encryption. 
+
+        Adding ``ENCRYPTION=NO`` to either a ``CREATE TABLE`` or ``ALTER TABLE`` statement results in a warning.
+
+The `CREATE TABLESPACE <https://dev.mysql.com/doc/refman/5.7/en/create-tablespace.html>`_ statement is extended to allow the ``ENCRYPTION=['
+Y/N']`` option.
+
+.. code-block:: guess
+
+    mysql> CREATE TABLE t1 (id INT) ENCRYPTION='Y';
+
+To encrypt an existing table, add the `ENCRYPTION` option in the ``ALTER TABLE`` statement. 
+
+.. code-block:: MySQL
+
+    mysql> ALTER TABLE t1 ENCRYPTION='Y';
+
+You can also disable encryption for a table, set the
+encryption to `N`.
+
+.. code-block:: MySQL
+
+    mysql> ALTER TABLE t1 ENCRYPTION='N';
+
+.. note::
+
+    The ``ALTER TABLE`` statment modifies the current encryption mode only if
+    the ``ENCRYPTION`` clause is explictily added.
+    
+.. rubric:: System Variables
+
+.. variable:: innodb_encrypt_tables
+
+   :version 5.7.21-21: Implemented
+   :cli: ``--innodb-encrypt-tables``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default: ``OFF``
+
+:Availability: This feature is **Experimental** quality.
```

Comment:
The feature is **Experimental** quality - does it refer to innodb_encrypt_tables or Encrypting Binary Logs. I assume the former.

Reply:
Fixed - changed the sentence to "This variable"